src/main/java/tr/com/srdc/cda2fhir/transform/CCDTransformerImpl.java

```diff
@@ -245,7 +246,8 @@ public Bundle transformDocument(ContinuityOfCareDocument cda) {
 	 * @param cda                A
 	 *
 	 *                           Consolidated CDA (C-CDA) 2.1 Continuity of Care
-	 *                           Document (CCD) instance to be transformed
```

Comment:
you've got a little weirdness from a merge conflict here, should take you like 2 seconds to fix.

Reply:
got it

Reply:
got it